docs/source/compilation.rst

```diff
@@ -0,0 +1,69 @@
+.. Copyright 2018 The Cartographer Authors
+
+.. Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+..      http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+==========================
+Compiling Cartographer ROS
+==========================
+
+System Requirements
+===================
+
+The Cartographer ROS requirements are the same as :ref:`the ones from Cartographer <cartographer:system-requirements>`.
```

Comment:
I'm not sure whether that `cartographer:system-requirements` link will work as it links to the external page of Cartographer?